code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -460,5 +463,113 @@ datum/species/harpy
 		"Your overheated skin itches."
 		)
 
-/datum/species/human/vatgrown
-	spawn_flags = SPECIES_IS_RESTRICTED
+/datum/species/crew_shadekin
+	name = SPECIES_SHADEKIN_CREW
+	name_plural = "Black-Eyed Shadekin"
+	icobase = 'icons/mob/human_races/r_shadekin_vr.dmi'
+	deform = 'icons/mob/human_races/r_shadekin_vr.dmi'
+	tail = "tail"
+	icobase_tail = 1
+	blurb = "Very little is known about these creatures. They appear to be largely mammalian in appearance. \
+	Seemingly very rare to encounter, there have been widespread myths of these creatures the galaxy over, \
+	but next to no verifiable evidence to their existence. However, they have recently been more verifiably \
+	documented in the Virgo system, following a mining bombardment of Virgo 3. The crew of NSB Adephagia have \
+	taken to calling these creatures 'Shadekin', and the name has generally stuck and spread. "		//TODO: Something more fitting for black-eyes
+	wikilink = "https://wiki.vore-station.net/Shadekin"
+	catalogue_data = list(/datum/category_item/catalogue/fauna/shadekin)
+
+	language = LANGUAGE_SHADEKIN
+	name_language = LANGUAGE_SHADEKIN
+	species_language = LANGUAGE_SHADEKIN
+	secondary_langs = list(LANGUAGE_SHADEKIN)
+	num_alternate_languages = 3
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws, /datum/unarmed_attack/bite/sharp)
+	rarity_value = 5	//INTERDIMENSIONAL FLUFFERS
+
+	siemens_coefficient = 0
+	darksight = 10
+
+	slowdown = 1.1
+	item_slowdown_mod = 1.5
+
+	total_health = 75
+	brute_mod = 1.5 // Frail
+	burn_mod = 1.5	// Furry
```

Comment:
I would say this is probably a bit **too** frail, and basically makes them slow, large Teshari.

I would personally recommend 100% health, 1.5 Brute, 1.25 Burn. Though Soft can say otherwise in this.